packages/react/src/Trans.test.tsx

```diff
@@ -54,13 +54,27 @@ describe("Trans component", function () {
     console.error = originalConsole
   })
 
-  it("should throw a console.error if using twice props", function () {
+  it("should log a console.error if using `render` and `component` props at the same time", function () {
+    const RenderChildrenInSpan = ({ children }: TransRenderProps) => (
+      <span>{children}</span>
+    )
+
     const originalConsole = console.error
     console.error = jest.fn()
```

Comment:
there is a special function `mockConsole` is `jest-mocks` package in monorepo. You can use it to make code more unified.